src/dos/drive_local.cpp

```diff
@@ -32,6 +32,12 @@
 #include "support.h"
 #include "cross.h"
 #include "inout.h"
+#include <sys/types.h>
+#ifdef _MSC_VER
+#include <sys/utime.h>
+#else
+#include <utime.h>
+#endif
```

Comment:
Please move these includes just after line 28, then keep empty line and project-specific includes after that.

In the past we needed to deal with pretty nasty linkage issues caused by incorrect include order, now we follow Google's style rules for include order in all new code, as it prevents those problems.